### DISCUSSION TITLE
Unraid support?

### DISCUSSION THREAD

**Discussion by Dalarialus** [nestybox/sysbox#565] [Q&A]
Hello! I was wondering if anyone has attempted to build/run sysbox on Unraid? I believe it is Slackware-based, and notice that Slack isn't on the supported OS list.

Many thanks!

**Comment by ctalledo** [comment]
Hi @Dalarialus,

Thanks for your interest in Sysbox.

I am not aware of anyone having attempted to run Sysbox on Unraid/Slackware. It's not officially supported, but my suggestion if you want to try it is to clone the Sysbox repo and [build Sysbox from source](https://github.com/nestybox/sysbox/blob/master/docs/developers-guide/build.md). Since the build occurs inside a Docker container itself (aka the "build container"), you will likely need to add a Dockerfile that sets up the build container with Unraid (similar to the Dockerfiles for other distros [here](https://github.com/nestybox/sysbox/tree/master/tests)).

We can help answer any questions that arise (as time allows).

Cheers!
